systemtest/datastreams_test.go

```diff
@@ -25,13 +25,14 @@ import (
 	"testing"
 	"time"
 
+	"github.com/elastic/apm-server/systemtest/estest"
```

Comment:
can you move this back please? all `github.com/elastic/...` import should be grouped together
did your IDE do this?

Reply:
It must have done, `make fmt` does not pick up on this ordering. Will see if GoLand can be configured to order imports like we dictate.